omero/developers/index.txt

```diff
@@ -115,6 +115,7 @@ Web
     Web/WritingViews
     Web/WritingTemplates
     Web/CSRF
+    Web/Webclient
```

Comment:
Ok, I guess that explains keeping it last, I hadn't thought of that. I think keeping this under Web/ is fine though.